test/bin/ci_phase_iso_boot.sh

```diff
@@ -27,6 +27,12 @@ cd ~/microshift/test
 # repository.
 bash -x ./bin/start_webserver.sh
 
+# set up the storage pool for VMs
```

Comment:
I've been trying to retain the ability for a developer to run some setup steps and then run 1 scenario, since it is likely impossible to run all of the scenarios at once on a laptop. We should land this as-is if it works, but then move it to a separate script so we can have a "configure" phase. The ci build phase script works like that so far, but logically these steps don't make sense there.

Reply:
Done.